services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/standards_standards_progress_report.jsx

```diff
@@ -47,43 +49,50 @@ export default class StandardsProgressReport extends React.Component {
         accessor: 'standard_level',
         sortType: sortTableByStandardLevel,
         resizable: false,
-        width: 150
+        maxWidth: 150
       }, {
         Header: "Standard name",
         accessor: 'standard_name',
         sortType: sortTableByStandardLevel,
-        minWidth: 200,
+        minWidth: 500,
         resizable: false,
-        Cell: ({row}) => (
-          <a className='row-link-disguise underlined' href={`/teachers/progress_reports/standards/classrooms/0/standards/${row.original['id']}/students`}>
-            {row.original['standard_name']}
-          </a>
-        )
+        Cell: ({ row }) => {
+          const { original } = row
+          const { id, name, standard_students_href } = original
+          return renderTooltipRow({ id, label: name, link: standard_students_href, headerWidth: 500 })
```

Comment:
Are headerWidth and minWidth 500 coupled? If so, I would extract as a constant to make that dependency explicit.